tests/security/apparmor/aa_status.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: apparmor tools testing - aa-status
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36874, tc#1621138
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    systemctl('restart apparmor');
```

Comment:
Why do you need to restart apparmor?

Reply:
It is not necessary, but when it is run with other cases, restart apparmor is a guarantee.

Reply:
A guarantee of what?

Reply:
In some installation, apparmor will not start automatically by default. (which is the case in my development environment)